bin/build_all

```diff
@@ -237,10 +237,10 @@ docker_build() {
 build_external() {
     docker_build docker/apache-basic_auth_proxy docker/ apache-basic_auth_proxy
     docker_build docker/grafana/sawtooth-stats-grafana \
-        docker/grafana \
+        . \
```

Comment:
This change seems unnecessary.  Is it?

Reply:
The build context has to be changed so that the grafana and influxdb startup scripts can be accessed from docker-compose. But it can be improved to just be `docker/`, which is the same format as the other build scripts. I will make this change